cms/bip/layouts/homepage/content.html

```diff
@@ -13,4 +13,9 @@
   {{ cms:partial:cms/homepage/meas }}
   {{ cms:partial:cms/homepage/themes }}
   {{ cms:partial:cms/homepage/indicators }}
+  <div class="u-hide">
+    {{ cms:page:newsletter_title:rich_text }}
```

Comment:
I would set the title just to text not rich_text as I don't think this functionality will be needed and rich text often puts a p-tag around the content, which is currently disrupting the header styling.

Reply:
Do you know what we can change this to? I have tried changing to just :text but still have the same issue

Reply:
it is looking good for me with the simple text field, no p tags, I will merge this now and see how it looks like on staging